opentracing-api/src/main/java/io/opentracing/ThreadLocalScheduler.java

```diff
@@ -0,0 +1,70 @@
+package io.opentracing;
+
+/**
+ * ThreadLocalScheduler is a trivial Scheduler implementation that relies on Java's thread-local storage primitives.
+ *
+ * @see Scheduler
+ * @see Tracer#scheduler()
+ */
+public class ThreadLocalScheduler implements Scheduler {
```

Comment:
@devinsba I see that point and don't disagree with it. That said, I'm not really concerned with the migration for existing users as much as ease-of-use for future users. I am still in favor of putting ThreadLocalScheduler in the core `opentracing-api` artifact, though perhaps (?) it should be in its own package. Thoughts?